src/components/Map.js

```diff
@@ -19,13 +25,27 @@ const Map = ({
     ).then((res) => setNearbyPlaces(res));
   }, [mapProperties.center.lat, mapProperties.center.lng, setNearbyPlaces]);
 
+  function handleApiLoaded(map, maps) {
+    setMapsObj(map, maps);
+  }
+
+  // only runs if maps object is populated
+  if (!isObjEmpty(mapsObj)) {
+    const { map } = mapsObj;
+    map.addListener("center_changed", function () {
+      setUserHasPanned(true);
+    });
+  }
+
   return (
     <div style={{ height: "calc(66.67vh - 1.25rem)", width: "100%" }}>
       <GoogleMapReact
         bootstrapURLKeys={{ key: "AIzaSyA_jF-TPUl8qTMZ3BKFTrFOolH9wR7NOz4" }}
         center={mapProperties.center}
         zoom={mapProperties.zoom}
         options={{ clickableIcons: false }}
+        yesIWantToUseGoogleMapApiInternals
```

Comment:
Does this prop not need anything assigned to it (like a bool)?

Reply:
I don't believe so. Based on the example in the docs, it's just dropped straight in.